_dev/sources/miscBuilder.ts

```diff
@@ -24,30 +30,78 @@ export class MiscBuilder {
         Here: "HERE Maps",
         MapBox: "MapBox"
     };
+    private purgeSettings: any;
+    private emailSenderFrom: string;
+    private customerClassification: string;
+    private isMarketplacePurchasesAllowed;
```

Comment:
Add types  to the next 3 lines

Reply:
ok. was following your strategy with isUnsignedAddinsAllowed but done.